xml/admin_operating_pools.xml

```diff
@@ -890,23 +890,47 @@ pool4                     1.1 MiB      13      0     39                  0
    need to migrate the pool data to another one whose parameters suit your
    deployment.
   </para>
-
   <para>
-   There are several methods of pool migration. We recommend using
-   <emphasis>cache tier</emphasis> because this method is transparent, reduces
-   cluster downtime, and avoids duplicating the entire pool's data.
+   There are two migration methods&mdash;a <emphasis>cache tier</emphasis>
```

Comment:
I think "there are" more than two methods. But only two of them are described here. So, probably change to something like "Thi document describes two migration methods ..."?